cohere/client.py

```diff
@@ -562,7 +566,7 @@ def _request(self, endpoint, json=None, method="POST", stream=False) -> Any:
             "Request-Source": self.request_source,
         }
 
-        url = f"{self.api_url}/{self.api_version}/{endpoint}"
+        url = f"{self.api_url}/{endpoint}"
```

Comment:
Find a proper solution for this one. This breaks all the other endpoints. BlobheartAPI has no concept of api versioning.